try.html

```diff
@@ -16,7 +16,7 @@
     If you like one, you can find <a href="/install">installation instructions here</a>.
 
 kernels: |
-    Jupyter kernels allow you to use Jupyter interfaces and tools with <bold>any programming language</bold>.
+    Jupyter kernels allow you to use Jupyter interfaces and tools with any programming language.
```

Comment:
The issue is it should be `<b>` not `<bold>`. I think it would be nice to have the emphasis as it was originally intended - we can cut it later if we decide otherwise.

```suggestion
    Jupyter kernels allow you to use Jupyter interfaces and tools with <b>any programming language</b>.
```